personalhandbok.md

```diff
@@ -73,40 +120,97 @@ Regnestykket blir da som følger:
+
 ### Bredbånd hjemmekontor
+I Array dekker vi mobil og bredbånd. Vi oppfordrer til å være nøktern, og oppleves det som at en ber om refusjon for urimelig dyre tjenester så kan vi holde tilbake deler av summen.
+
 ### Reiseregninger / refusjon egne utlegg
+Reiseregninger og refusjoner tas på den administrative Slack-kanalen, eller med daglig tjener.
+
 ### Lunsj-/kantineutgifter
+Vi har for øyeblikket ingen lønsjordning, men den er på vei. Vi ønsker at alle skal betale inn kr. 500, og deretter få dekket lønsjen sin uansett om de er ute hos kunde eller på kontoret vårt.
+
 ### Refusjon av "diverse utlegg"/sosiale begivenheter
+Som et utgangspunkt er vi rause med å dekke mat og drikke i forbindelse med sosiale arrangementer med oss selv eller våre kunder. Normalt sett er det innenfor å levere inn kvitteringer for ulike sosiale formål. Det gjøres på den administrative Slack-kanalen.
+
+Er det tvil om hva som er innenfor eller ikke så kan en spørre :-)
+
 ### Betingelser for utgiftsrefusjon
+Utstyr du har behov for må du enten kjøpe selv og levere inn kvittering for. Er det noe du er i tvil om du får dekket, så tar du kontakt med daglig tjener eller spør på Slack. Alle har informasjonsplikt om hva de har kjøpt, og vi er åpne om hvilket utstyr/gadgets vi har kjøpt inn.
+
+Andre typer utgifter du måtte ha kan du levere inn kvitteringer for dersom det er relatert til arbeidet ditt eller et sosialt arrangement i Array-regi.
+
+Det anbefales at en holder seg normale "Codes of conduct"
 
 ## Generell informasjon
-### Medarbeidersamtale
+### Medarbeider- og utviklingssamtale
+Alle ansatte skal ha medarbeidersamtale to ganger i året, oppfølgingssamtaler ikke medregnet. Disse samtalene skal dekke det som normalt er med for å sikre medbestemmelse, ansvarsområder, hvordan den ansatte utvikler seg i selskapet og så videre.
+
+Samtalen bør vare rundt en time og ha et fast format som ivaretar interessene til arbeidstaker.
+
 ### Permisjoner
+Ansatte kan få velferdspermisjon med lønn i inntil 12 arbeidsdager i året når det finnes viktige velferdsgrunner.
+
+* Alvorlig sykdom i arbeidstakers nærmeste familie
+* Dødsfall i nærmeste familie eller hos andre som har stått en nær
+* Permisjon til å delta i begravelse
+* Oppfølging av barn i forbindelse med innleggelse eller behandling ved helseinstitusjon
+* Dersom begge foreldre er i arbeid gis det permisjon til tilvenning av barn i ny barnehage/SFO/barnepasser i inntil tre dager. Retten gjelder for én av de foresatte, men dagene kan deles.
+* Én dags permisjon for å følge barn til første skoledag i grunnskolen
+* Dersom en gifter seg på en arbeidsdag gis det permisjon den dagen ekteskapet inngås.
+* Rund dag; Alle bursdager som slutter på 0 gir permisjon med lønn (men kom gjerne innom på kake)
```

Comment:
Begynte i Array 8 mnd for sent 😢

Reply:
Nå kan du angre! Holde ut i 9 år og 4 mnd så løser det seg skal du se!